src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -395,10 +393,11 @@ public int GroupNumberFromName(String name)
             // look up name if we have a hashtable of names
             if (_capnames != null)
             {
-                if (!_capnames.ContainsKey(name))
+                int result;
```

Comment:
This will not compile.  Please make sure you're building and testing locally before submitting any changes in a PR.  We appreciate your contributions, but this has been happening a fair amount, and it makes it difficult for us to continue to evaluate such changes.